include/grpc++/impl/codegen/status.h

```diff
@@ -48,10 +48,13 @@ class Status {
   Status() : code_(StatusCode::OK) {}
 
   /// Construct an instance with associated \a code and \a error_message
```

Comment:
Add a terminating period to the previous sentence? Same for the other comment.

I also wonder if we should enforce that with an assert.